stl/inc/ranges

```diff
@@ -6725,8 +6725,9 @@ namespace ranges {
         && constructible_from<_Container, _Rng, _Types...>;
 
     template <class _Rng, class _Container, class... _Types>
-    concept _Converts_tag_constructible = _Ref_converts<_Rng, _Container> //
-        && constructible_from<_Container, from_range_t, _Rng, _Types...>;
+    concept _Converts_tag_constructible = _Ref_converts<_Rng, _Container>
+        // per LWG issue unnumbered as of 2022-08-08
+        && constructible_from<_Container, const from_range_t&, _Rng, _Types...>;

```

Comment:
I noticed this issue recently too. 
And when I used MSVC-STL for resolution validation I found that you already dealt with it. Are you planning to file this LWG and `stride_view::iterator::operator+=` recently?